recipes/capnproto/all/conanfile.py

```diff
@@ -187,27 +198,35 @@ def package_info(self):
             components.append({"name": "kj-gzip", "requires": ["kj", "kj-async", "zlib::zlib"]})
         if self.options.with_openssl:
             components.append({"name": "kj-tls", "requires": ["kj", "kj-async", "openssl::openssl"]})
+        if tools.Version(self.version) >= "0.9.0":
+            components.append({
+                "name": "capnp-websocket",
+                "requires": ["capnp", "capnp-rpc", "kj-http", "kj-async", "kj"],
+            })
 
         for component in components:
             self._register_component(component)
 
-        if self.settings.os == "Linux":
+        if self.settings.os in ["Linux", "FreeBSD"]:
             self.cpp_info.components["capnpc"].system_libs = ["pthread"]
             self.cpp_info.components["kj"].system_libs = ["pthread"]
             self.cpp_info.components["kj-async"].system_libs = ["pthread"]
         elif self.settings.os == "Windows":
             self.cpp_info.components["kj-async"].system_libs = ["ws2_32"]
         self.cpp_info.components["kj"].builddirs.append(self._cmake_folder)
-        self.cpp_info.components["kj"].build_modules = [os.path.join(self._cmake_folder, "CapnProtoMacros.cmake")]
+        capnprotomacros = os.path.join(self._cmake_folder, "CapnProtoMacros.cmake")
+        self.cpp_info.components["kj"].set_property("cmake_build_modules", [capnprotomacros])
```

Comment:
may need to be set at global scope
see https://github.com/conan-io/conan/issues/10282